src/components/application_manager/src/commands/mobile/put_file_request.cc

```diff
@@ -38,6 +38,18 @@
 #include "application_manager/application_impl.h"
 
 #include "utils/file_system.h"
+#include <boost/crc.hpp>
+
+namespace {
+
+uint32_t GetCrc32CheckSum(const std::vector<uint8_t>& binary_data) {
+  std::size_t file_size = binary_data.size();
```

Comment:
@AByzhynar const variable

Reply:
@LuxoftAKutsan Fixed in 9e46887993433e22387c52b6995a39274581646b